src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task {
+    protected DateAndTime at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = new DateAndTime(at);
```

Comment:
What if the input given by the user is not an intended date and time? eg. (at: Block 903/904)?

Reply:
I have tried to, in my more recent commits, ensure the date and time is of the correct format. I throw an exception if it is not!